source/userguide/auto_annotator.rst

```diff
@@ -0,0 +1,474 @@
+
+		"annotate": [ 
+
+			{ 
+
+				"domains": "*", 
+
+				"intents": "*", 
+
+				"files": "*", 
+
+				"entities": "*", 
+
+			} 
+
+		], 
+
+		"unannotate_supported_entities_only": True, 
+
+		"unannotate": None, 
+	}
+
+Let's take a look at the allowed values for each setting in an Auto Annotator configuration.
+
+
+``'annotator_class'`` (:class:`str`): The class in auto_annotator.py to use for annotation. By default, :class:`SpacyAnnotator` is used. 
+
+``'overwrite'`` (:class:`bool`): Whether new annotations should overwrite existing annotations in the case of a span conflict. False by default. 
+
+``'annotate'`` (:class:`list`): A list of annotation rules where each rule is represented as a dictionary. Each rule must have four keys: :attr:`domains`, :attr:`intents`, :attr:`files`, and :attr:`entities`.
+Annotation rules are combined internally to create Regex patterns to match selected files. :attr:`*` can be used if all possibilities in a section are to be selected. :attr:`|` is used to represent “or”.  
+Let's take a look at an example annotation rule:
+
+.. code-block:: python
+
+	{
+
+		"domains": "faq|salary", 
+
+		"intents": "*", 
+
+		"files": "train.txt|test.txt", 
+
+		"entities": "sys_amount-of-money|sys_time", 
+
+	}
+
+The rule above would annotate all text files named "train" or "test" in the "faq" and "salary" domains. Only sys_amount-of-money and sys_time entities would be annotated.
+
+.. note::
+
+	The order of the annotation rules matters. A rule will overwrite a previous rule if there are conflicts.
+
+.. warning::
+	By default, all files in all intents across all domains will be annotated with all supported entities. Before annotating consider including custom annotation rules in :attr:`config.py`. 
+
+``'unannotate_supported_entities_only'`` (:class:`boolean`): By default, when the unannotate command is used only entities that the Annotator can annotate will be eligible for removal. 
+
+``'unannotate'`` (:class:`list`): List of annotation rules in the same format as those used for annotation. These rules specify which entities should have their annotations removed. By default, :attr:`files` is None.
+
+``'spacy_model'`` (:class:`str`): :attr:`en_core_web_lg` is used by default for the best performance. Alternative options are :attr:`en_core_web_sm` and :attr:`en_core_web_md`. This parameter is optional and is specific to the use of the :class:`SpacyAnnotator`.
+If the selected english model is not in the current environment it will automatically be downloaded. Refer to Spacy's documentation to learn more about their `english models <https://spacy.io/models/en>`_.
+
+
+Using the Auto Annotator
+------------------------
+
+The Auto Annotator can be used by importing a class that implements the :class:`Annotator` abstract class in the :mod:`auto_annotator` module or through the command-line.
+We will demonstrate both approaches for Unannotation and Annotation using the :class:`SpacyAnnotator` class.
```

Comment:
That's correct, a more natural workflow is to annotate and then unannotate. While that makes more sense in practice, since blueprints are already annotated it's easier to demonstrate in this order. I have added this sentence to help clarify what the natural workflow is: 

"For the this tutorial we will first describe unannotation and then annotation, however, in a normal workflow you are likely to annotate first and then unannotate any annotations you are not pleased with."